locust/contrib/fasthttp.py

```diff
@@ -578,7 +578,10 @@ class ResponseContextManager(FastResponse):
     def __init__(self, response, environment, request_meta):
         # copy data from response to this object
         self.__dict__ = response.__dict__
-        self._cached_content = response.content
+        try:
+            self._cached_content = response._cached_content
+        except AttributeError:
+            pass
```

Comment:
I've also improved the additional test just now.

Reply:
I dont understand it either. I was thinking it is better to have self._cached_content set to None rather than having it completely undefined, but maybe it is not.

But I dont think you're breaking anything by adding the try-catch so if you are good to go, I can merge this.

Reply:
@cyberw Yes, please merge. Any plan to release it also? My `locustfile.py` is waiting for that upstream fix :) Also, thanks for maintaining this project so well. Your response time is awesome!